plugins/nid-team/rebase/rebase_manager.py

```
@@ -0,0 +1,702 @@
+                    # Let's inspect changed files to see if config-only
+                    try:
+                        changed_files = self.run_cmd(["git", "show", "--name-only", "--pretty=format:", sha]).split("\n")
+                        config_only = all(
+                            any(x in f for x in ["Dockerfile", "OWNERS", ".ci-operator.yaml", ".snyk", "Makefile", "rebase"])
+                            for f in changed_files if f.strip()
+                        )
+                        if config_only:
+                            decision = "squash"
+                            reason = "Downstream configuration change (candidates for squashing into core files carry)"
+                    except Exception:
+                        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)